docs/contributing.rst

```diff
@@ -85,3 +85,126 @@ The code style is enforced by `flake8 <https://pypi.org/project/flake8>`_, its
 configuration, and possibly extensions. No code style review should be done by
 a human. All code style enforcement must be automatized to avoid bikeshedding
 and losing time.
+
+
+How-to: Write an Integration
+============================
+
+First off, thank you for writing a new integration! The best way to get started
+writing a new integration is to refer to existing integrations, preferably one
```

Comment:
we might want to have this guide be the defacto example of how to write an integration instead of telling them to look at something existing.

what if `redis` becomes the old way of doing things?